crates/ruff_workspace/src/options.rs

```diff
@@ -400,10 +418,14 @@ pub struct Options {
         default = "4",
         value_type = "int",
         example = r#"
-            tab-size = 8
+            tab-size = 2
         "#
     )]
-    pub tab_size: Option<TabSize>,
+    #[deprecated(
+        since = "0.1.2",
+        note = "The `tab-size` option has been renamed to `indent-width` to emphasize that it configures the indentation used by the formatter as well as the tab width. Please update your configuration to use `indent-width = <value>` instead."
```

Comment:
Same question as in the parent PR - Do we want to provide the version in which the option will be removed?

Reply:
I don't think I've a good enough intuition for our release cycle yet to predict a good depreciation version, and I want to avoid that I reference a version that might never ship (we may end up releasing 1.0 before we ever ship 0.5)